src/test/java/com/google/sps/BigBusinessesTest.java

```diff
@@ -0,0 +1,146 @@
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.util.List;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.ArrayList;
+import java.util.Iterator;
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
+import com.google.maps.model.Photo;
+
+@RunWith(JUnit4.class)
+public final class BigBusinessesTest {
+  private Photo[] samplePhotos = new Photo[0];
+  private String[] sampleBusinessTypes = new String[0];
+  private MapLocation googlesOffice = new MapLocation(40.457177, -79.916696);
+  private DatastoreService datastore;
+  private List<Listing> sampleListOfBusinesses = new LinkedList<Listing>(Arrays.asList(
+                new Listing("LA Fitness",
+                  "Address", 
+                  new MapLocation(40.457091, -79.915331), 
+                  3.9,
+                  samplePhotos, 
+                  sampleBusinessTypes),
+                new Listing("west elm", 
+                  "Address",
+                  new MapLocation(40.456279, -79.915015), 
+                  3.6, 
+                  samplePhotos,
+                  sampleBusinessTypes),
+                new Listing("McDonalds",
+                  "Address", 
+                  new MapLocation(40.459450, -79.918479), 
+                  2.6, 
+                  samplePhotos,
+                  sampleBusinessTypes),
+                new Listing("East End Brewing Company", 
+                  "Address",
+                  new MapLocation(40.459391, -79.911782), 
+                  4.7,
+                  samplePhotos, 
+                  sampleBusinessTypes),
+              new Listing("The Shiny Bean Coffee & Tea", 
+                  "Address",
+                  new MapLocation(40.496328, -79.944862), 
+                  4.9, 
+                  samplePhotos,
+                  sampleBusinessTypes),
+                new Listing("Weisshouse", 
+                  "Address",
+                  new MapLocation(40.456684, -79.925499), 
+                  4.3,
+                  samplePhotos, 
+                  sampleBusinessTypes)
+  ));  
+  private List<Listing> expectedListOfBusinesses = new LinkedList<>();  
```

Comment:
nit: Please add a line above this statement.

Reply:
fixed